examples/real-world/.babelrc.js

```diff
@@ -9,7 +9,6 @@ module.exports = {
       '@babel/env',
       {
         loose: true,
-        modules: false,
         forceAllTransforms: true,
```

Comment:
why the change here? this demo uses webpack and it's better not to transpile modules when using it

Reply:
`server.js` is not built with webpack. second option is use CJS for server.js